cibuildwheel/resources/install_certifi.py

```diff
@@ -0,0 +1,44 @@
+# Based on: https://github.com/python/cpython/blob/master/Mac/BuildScript/resources/install_certificates.command
+
+# install_certifi.py
+#
+# sample script to install or update a set of default Root Certificates
+# for the ssl module.  Uses the certificates provided by the certifi package:
+#       https://pypi.org/project/certifi/
+import os
```

Comment:
New line, before?

Reply:
I copy file content, I forget to reformat.